<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShoeClient</class>
 <widget class="QWidget" name="ShoeClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShoeClient</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_panel">
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_sendpacket">
       <item>
        <widget class="QPushButton" name="pushButton_login">
         <property name="text">
          <string>模拟登陆</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_gps">
         <property name="text">
          <string>模拟GPS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_clear">
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_connect">
     <item>
      <widget class="QLabel" name="label_ip">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>服务器地址：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_ip">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>47.93.6.161</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>192.168.1.110</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>192.168.1.114</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_port">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>端口号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_port">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>7008</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_connect">
       <property name="text">
        <string>连接</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
